src/screens/SUE/SueReportScreen.tsx

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable complexity */
```

Comment:
we place this in front of the code that is causing and not on top of files, right?

Reply:
Yes, that's right. Actually, I don't think it's necessary, because the code is not that complex.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/966/commits/2b603bd8f6f8b43bf449efd90fce208da3b2c23c